yal/askaquestion.py

```diff
@@ -254,26 +255,31 @@ async def state_display_content(self):
         return await self.go_to_state("state_get_content_feedback")
 
     async def state_get_content_feedback(self):
+        choices = [
+            Choice("yes", self._("Yes")),
+            Choice("back to list", self._("No, go back to list")),
+            Choice("no", self._("Nope...")),
+        ]
         question = self._(
             "\n".join(
                 [
-                    "*Did I answer your question?*",
+                    "[persona_emoji] *Did I answer your question?*",
                     "",
                     "*Reply:*",
-                    "*1* - Yes",
-                    "*2* - No, go back to list",
+                    get_display_choices(choices, bold_numbers=True),
                 ]
             )
         )
         return WhatsAppButtonState(
             self,
             question=question,
-            choices=[
-                Choice("yes", "Yes"),
-                Choice("no", "No"),
-            ],
+            choices=choices,
             error=self._(get_generic_error()),
-            next="state_is_question_answered",
+            next={
+                "yes": "state_is_question_answered",
+                "back to list": "state_display_results",
+                "no": "state_no_question_not_answered",
```

Comment:
the "no" response must also please go to `state_is_question_answered` first.
This is so that we can send the positive or negative feedback back to the model
So the bottom of `state_is_question_answered` should direct to `state_no_question_not_answered` instead

Reply:
`back to list` should also go to state_is_question_answered` for the same reason. you can do this split based on the users input at the end of that